frontend/appflowy_flutter/lib/workspace/presentation/home/home_screen.dart

```diff
@@ -287,3 +305,62 @@ class HomeScreenStackAdaptor extends HomeStackDelegate {
     });
   }
 }
+
+class WindowButtons extends StatefulWidget {
+  const WindowButtons({Key? key}) : super(key: key);
+
+  @override
+  WindowButtonsState createState() => WindowButtonsState();
+}
+
+class WindowButtonsState extends State<WindowButtons> {
+  late WindowButtonColors buttonColors;
+  late WindowButtonColors closeButtonColors;
+
+  void maximizeOrRestore() {
+    setState(() {
+      appWindow.maximizeOrRestore();
+    });
+  }
+
+  @override
+  void didChangeDependencies() {
+    buttonColors = WindowButtonColors(
+        normal: Colors.transparent,
+        iconNormal: Theme.of(context).colorScheme.primary,
+        mouseOver: const Color(0xFF404040),
```

Comment:
These values should be read from our theme, not hardcoded here. If there are not in the theme, please add them as you see fit. Somewhere here: frontend/appflowy_flutter/packages/flowy_infra/lib/theme.dart